### ISSUE TITLE
Synchronous reads

### ISSUE THREAD

**Issue by karimi68** [broughtong/PyMercuryLib#3]
Can this library work in synchronous read mode?

**Comment by MFernandezCarmona** [comment]
No, by now it only supports asynchronous continuous reading. But Mercury RFID reader library has a method (TMR_read) that supports this synchronous reading mode... but we haven't made the python wrapper yet.
